modules/Hoot/managers/api.js

```diff
@@ -1540,4 +1529,127 @@ export default class API {
 
         return this.request( params );
     }
+
+    /**
+     * Retrieves all the projects from tasking manager
+     */
+    getTMProjects() {
+        const params = {
+            url: '/tasks/projects.json',
+            method: 'GET',
+            headers: {
+                'X-Requested-With': 'XMLHttpRequest'
+            }
+        };
+
+        return this.request( params )
+            .then( resp => resp.data )
+            .catch( () => {
+                let alert = {
+                    message: 'Failed to retrieve projects list from tasking manager.',
+                    type: 'error'
+                };
+
+                Hoot.message.alert( alert );
+            } );
+    }
+
+    /**
+     * Retrieves all the tasks from tasking manager for the specified project
+     */
+    getTMTasks( projectId ) {
```

Comment:
Do you think we should poll this when the TM tab is active?  I noticed that the TM UI will refresh when Hoot locks a task so maybe Hoot should refresh if a user locks a task in TM?

Reply:
The 1 problem I see right now is that it would ruin the 'Running' status since that is something I added and it isn't a part of TM of course